docs/topics/http/urls.txt

```diff
@@ -146,20 +146,25 @@ A converter is a class that includes the following:
 
 * A ``to_python(self, value)`` method, which handles converting the matched
   string into the type that should be passed to the view function. It should
-  raise ``ValueError`` if it can't convert the given value. A ``ValueError`` is
-  interpreted as no match and as a consequence a 404 response is sent to the
+  raise ``Http404`` if it can't convert the given value. A ``Http404`` is
+  raised as no match and as a consequence a 404 response is sent to the
```

Comment:
```
raise ``ValueError`` or ``Http404`` if it can't convert the given value. A
``ValueError`` is interpreted as no match and as a consequence a 404
response is sent to the user.
```